dags/shredder.py

```diff
@@ -95,6 +95,21 @@
         "--parallelism=2",
         "--billing-project=moz-fx-data-bq-batch-prod",
         "--only=telemetry_derived.main_summary_v4",
+        "--no-use-dml",
+        # temporarily override parallelism and billing projects to use backfill projects
+        # to spread jobs out and increase speed of sanitizing columns for Bug 1751979
+        "--parallelism=10",
```

Comment:
We have `         "--parallelism=2",` a few lines above this. We should make sure to remove that line.